common/lib/capa/capa/capa_problem.py

```diff
@@ -561,22 +561,19 @@ def find_question_label(self, answer_id):
             a string with the question text
         """
 
-        def log_error():
-            """
-            Temporary method for logging
-            """
-            log.error(
-                'KeyError: answer_id: {}, Problem data: {}, problem: {}'
-                .format(answer_id, self.problem_data, etree.tostring(self.tree))
-            )
+        def generate_default_question_label():
```

Comment:
This method needs a docstring.

Reply:
Add a one-liner purpose docstring before the in-depth explanation part. Something like `Add a default question label if not present in the text` etc.